docs/UserGuide.md

```diff
@@ -507,6 +524,52 @@ Format: `exit`
 
 --------------------------------------------------------------------------------------------------------------------
 
+### 2.7. Quick View Panel
+
+The Quick View panel allows you to see general details of your guests at a glance. It is located on the left side of the window.
+
+<img src="images/quick-view-panel.png" alt="WedLog's Quick View panel" height="500"> <br />
+
+The Quick View panel consists of 2 sections: the RSVP Status panel and the Dietary Requirements panel.
+
+--------------------------------------------------------------------------------------------------------------------
+
+#### 2.7.1. RSVP Status panel
+
+The RSVP Status panel is designed to help you keep track of the proportion of RSVP statuses of your guests. Representing the
+data is a pie chart, with labels containing the number of guests with each RSVP status. The pie chart is updated automatically
+whenever you add, delete or edit a guest.
+
+<img src="images/rsvp-status-panel.png" alt="RSVP Status panel"> <br />
+
+The pie chart slices are colour-coded as follows:
+- `Yes`: Green
+- `No`: Red
+- `Unknown`: Orange
+
+--------------------------------------------------------------------------------------------------------------------
+
+#### 2.7.2. Dietary Requirements panel
+
+The Dietary Requirements panel is designed to help you keep track of your guests' dietary requirements. Whenever you specify
+a guest to have certain dietary requirements, the specific requirements, will be added to the panel, together with
+a count of the number of guests with the same requirements. This panel consolidates your guests' dietary requirements to be displayed together, 
+for you to consider when planning the menu for your wedding. The panel is also updated automatically whenever you add, delete or edit a guest.
+
+<img src="images/dietary-requirements-panel.png" alt="Dietary Requirements panel"> <br />
+
+Things to note:
+- **The panel only displays dietary requirements for guests with RSVP status "yes".** This is because guests with RSVP status "no" or "unknown" 
```

Comment:
```suggestion
- **The panel only displays dietary requirements for guests with RSVP status `Yes`.** This is because guests with RSVP status `No` or `unknown`
```

Reply:
For standardization with earlier parts of the UG.